system/lib/compiler-rt/lib/sanitizer_common/sanitizer_posix.cc

```diff
@@ -279,6 +290,10 @@ void ReportFile::Write(const char *buffer, uptr length) {
 }
 
 bool GetCodeRangeForFile(const char *module, uptr *start, uptr *end) {
+#if SANITIZER_EMSCRIPTEN
+  // Code is not mapped in memory in Emscripten, so this operation is meaningless
+  // and thus always fails.
```

Comment:
Would it make more sense to always succeed with an empty range?

Reply:
I am more inclined to make this an error. We don't know what this is used for yet, so I'd rather not make it succeed.